src/ContractInteractor.ts

```diff
@@ -540,6 +565,7 @@ export default class ContractInteractor {
         });
 
         // TODO RIF Team: Once the exactimator is available on the RSK node, then ESTIMATED_GAS_CORRECTION_FACTOR can be removed (in our tests it is 1.0 anyway, so it's not active)
+        //FIXME check why two times its being multiply by ESTIMATED_GAS_CORRECTION_FACTOR
```

Comment:
Is it still valid? Are we planning to fix it in this task or do we need to create a new task for this?

Reply:
fixed in the relayClient